qubrabench/algorithms/search.py

```diff
@@ -81,6 +83,7 @@ def cade_et_al_expected_quantum_queries(N: int, T: int, eps: float, K: int) -> f
     Returns:
         the upper bound on the number of quantum queries
     """
+    N = float(N)
```

Comment:
Because
```
>>> np.sqrt(1000000000000000000000000000000000000)
AttributeError: 'int' object has no attribute 'sqrt'
```
(see discussion on Slack).